.teamcity/src/builds/Lint.kt

```diff
@@ -12,12 +12,12 @@ object Lint : BuildType({
 
```

Comment:
nit: also update the task description to replace sasslint with stylelint.

Reply:
pushed with b4d9ab8